galaxy/tools/deps/mulled/mulled_build.py

```diff
@@ -325,10 +322,8 @@ def add_build_arguments(parser):
                         help='quay.io namespace.')
     parser.add_argument('-r', '--repository_template', dest='repository_template', default=DEFAULT_REPOSITORY_TEMPLATE,
                         help='Docker repository target for publication (only quay.io or compat. API is currently supported).')
-    parser.add_argument('-c', '--channel', dest='channel', default=DEFAULT_CHANNEL,
-                        help='Target conda channel')
-    parser.add_argument('--extra-channels', dest='extra_channels', default=",".join(DEFAULT_EXTRA_CHANNELS),
-                        help='Dependent conda channels.')
+    parser.add_argument('-c', '--channel', dest='channels', default=DEFAULT_CHANNELS,
```

Comment:
What about `--channels` instead of `--channel` ? Or it's not worth the possible breakage?